NEWS.md

```diff
@@ -18,120 +18,161 @@
 MXNet Change Log
 ================
 - [MXNet Change Log](#mxnet-change-log)
-  * [1.6.0](#160)
-    + [Deprecation of Python 2](#deprecation-of-python-2)
-    + [New Features](#new-features)
-      - [NumPy compatible interface and using TVM to generate operators](#numpy-compatible-interface-and-using-tvm-to-generate-operators)
-      - [Graph optimizations](#graph-optimizations)
+  - [1.7.0](#170)
+    - [New features](#new-features)
+      - [MXNet Extensions: custom operators, partitioning, and graph passes](#mxnet-extensions-custom-operators-partitioning-and-graph-passes)
+      - [OpPerf utility enabled in the binary distribution](#opperf-utility-enabled-in-the-binary-distribution)
```

Comment:
which PR did this?

Reply:
Hi @ChaiBapchya , probably it's not drived by any PR in the current scope, but included in the binary release process. cc @szha to see if any comments.
When trying to install mxnet release (1.7.0) wheel via pip, the benchmark/opperf is also installed now. And the original proposal from you is listed in [1.7.0 roadmap](https://github.com/apache/incubator-mxnet/issues/16864#issuecomment-585461851)